input/haleu/run_multiple_cyclus_inputs.bash

```diff
@@ -1,5 +1,5 @@
-for scenario in mmr xe100 xe100_mmr mmr_voygr xe100_voygr xe100_mmr_voygr 
+for scenario in 2 3 4 5 6 7 8 9 10 11 12 13
```

Comment:
okay, I see why you wanted to have this relatively simple naming convention. I think my suggestion still works, since the scenario number is contained. Since you're using a bash script, you can be a LOT more creative with how you're accessing files. 

E.g. save the files in your input folder as a list variable, then loop through each item in the list variable, rather than  a numerical range.

Reply:
Idk I'd also want a way for users to run specific scenarios interactively? I think you could do this. But it's not required.

Reply:
Users can always run a single input file by running `cyclus -i inputs/file names -o outputs/output_database.sqlite`. This script was a way for me to easy run them in series so that I didn't have to keep typing that.

Reply:
Sure, but what about a specific set of scenarios (e.g., 1,4,7)?

Reply:
Then they can run those in series from the command line or adjust this file to include only the scenarios they want to run.